### DISCUSSION TITLE
[@types/jquery] Type for $(selector) does not reflect that null can be returned

### DISCUSSION THREAD

**Discussion by jeremy-rifkin** [DefinitelyTyped/DefinitelyTyped#59860] [Issues with a @types package] [answered]
Current typing for @types/jquery indicates `$(selector)` returns `JQuery<TElement>` but if the selector is not found it returns null.

[a comment by an automated account is not shown]

**Comment by leonard-thieu** [comment, marked as answer]
> Current typing for @types/jquery indicates `$(selector)` returns `JQuery<TElement>` but if the selector is not found it returns null.

I'm getting a jQuery object with a non-existent selector.

https://jsfiddle.net/leonard_thieu/065ysq7u/6/

**Reply by jeremy-rifkin** [reply]
Thank you for the quick reply, my bad! I made a silly mistake while testing something locally. Thanks for taking the time!
